ggml/src/ggml-vulkan/vulkan-shaders/conv2d_mm.comp

```diff
@@ -225,10 +229,16 @@ void main() {
             uint32_t B_ly    = r_offset + Ar;
             uint32_t B_lx    = Ac;
             uint32_t K_idx   = B_idx_K * BS_K + B_ly; /* Global K_idx (row index of A)*/
-            uint32_t knl_idx = min(KW_idx_a + KH_idx_a * p.nb01 + Cin_idx_a * p.nb02 + K_idx * p.nb03, K * CRS - 1);
```

Comment:
The index calculation is moved into the else, is it indended? Some of such optimizations backfired on a few GPUs https://github.com/ggml-org/llama.cpp/pull/15056